rails-commenting.rb

```diff
@@ -6,57 +6,57 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) The controller is created in the Terminal, rails generate controller BlogPost would be the command. The controller allows for each of the CRUD routes to have a method. The 7 CRUD routes consist of Index, Show, Create, Edit, Update and Destroy. A model is passed through the controller, routes and views to establish an MVP which stands for minimum viable product.
 class BlogPostsController < ApplicationController
```

Comment:
Important to keep in mind that this controller is inheriting from ApplicationController.